Sources/Actions/Admin/Members.php

```diff
@@ -1453,22 +1453,15 @@ public static function list_getMembers(int $start, int $items_per_page, string $
 	 */
 	public static function list_getNumMembers(string $where, array $where_params = []): int
 	{
-		// We know how many members there are in total.
-		if (empty($where) || $where == '1=1') {
-			$num_members = Config::$modSettings['totalMembers'];
-		}
-		// The database knows the amount when there are extra conditions.
-		else {
-			$request = Db::$db->query(
-				'SELECT COUNT(*)
-				FROM {db_prefix}members AS mem
-				WHERE ' . $where,
-				array_merge($where_params, [
-				]),
-			);
-			list($num_members) = Db::$db->fetch_row($request);
-			Db::$db->free_result($request);
-		}
+		$request = Db::$db->query(
+			'SELECT COUNT(*)
+			FROM {db_prefix}members AS mem
+			WHERE ' . $where,
+			array_merge($where_params, [
+			]),
```

Comment:
```suggestion
			$where_params,
```

Not much point merging an empty array. 🙂